config/custom-plugin-monitor.json

```diff
@@ -15,12 +15,6 @@
     }
   ],
   "rules": [
-    {
-      "type": "temporary",
-      "reason": "NTPIsDown",
```

Comment:
Since each condition is followed by an event, this can be deleted as we have added `NTPIsDown` condition. But i prefer we add another different temporary event as an example. :)

Reply:
Will keep this as an example.

Reply:
Done